### PR TITLE
Add link to the eslint-config-vtex

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
- Add link to the `eslint-config-vtex`

#### Related to:
[Eslint-config-vtex PR](https://github.com/vtex/eslint-config-vtex/pull/15)

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by Caio-Batista — APPROVED** [review]
LGTM

File: index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  "extends": "vtex",
```
Comment (klzns) [inline index.js:2]: Acho que nao deveria extender ele, eles sao independentes.

File: package.json [hunk 2]
```diff
@@ -19,10 +19,11 @@
   "author": "Breno Calazans",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/brenoc/eslint-config-vtex-react/issues"
+    "url": "https://github.com/vtex/eslint-config-vtex-react/issues"
   },
-  "homepage": "https://github.com/brenoc/eslint-config-vtex-react#readme",
+  "homepage": "https://github.com/vted/eslint-config-vtex-react#readme",
```
Comment (klzns) [inline package.json]: typo

File: package.json [hunk 3]
```diff
@@ -19,10 +19,11 @@
...
   "dependencies": {
+    "eslint-config-vtex": "^8.0.0",
```
Comment (klzns) [inline package.json]: remover, caso concorde com o comentario acima

**Review by klzns — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (salesfelipe) [reply index.js:2]: Como discutimos anteriormente, como o objetivo é unificar o lint da vtex acho justo eles estarem linkados dado que uma app react tb é uma app javascript e por isso deveriam seguir as mesmas regras de lint.

File: package.json (hunk 2 above)
Comment (salesfelipe) [reply package.json]: 👍

**Review by klzns — APPROVED** [review]

**Pull request closed without merging**